portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.testcase

```diff
@@ -1,29 +1,118 @@
 @component-name = "portal-web-forms-and-data-lists"
 definition {
-property testray.main.component.name = "Forms";
-setUp {
-TestCase.setUpPortalInstance();
-User.firstLoginPG();
-Navigator.openURL();
-}
-tearDown {
-User.logoutAndLoginPG(
-userLoginEmailAddress = "test@liferay.com",
-userLoginFullName = "Test Test");
-}
-
-@description = "Access Portal Global Menu and validate CONFIGURATION string at
-Control Panel"
-@priority = "5"
-test ValidateConfigurationStringAtControlPanel {
-task("accessing the option menu"){
-        MyTestFile.clickMenu();
-}
-task("click on control panel option"){
-        MyTestFile.clickControPanel();
-}
-task("check the configuration section is present"){
-        MyTestFile.assertConfiguration();
-}
-}
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
```

Comment:
Look for example of tearDown in other form files, and check what best applies to these cases